Games/types/Mafia/roles/cards/ItemTransfer.js

```diff
@@ -0,0 +1,48 @@
+const Card = require("../../Card");
+const { PRIORITY_ITEM_GIVER_DEFAULT } = require("../../const/Priority");
+
+module.exports = class ItemTransfer extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Steal From": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive"], exclude: ["dead", "self"] },
+                action: {
+                    labels: ["stealItem"],
+                    priority: PRIORITY_ITEM_GIVER_DEFAULT - 5,
+                    run: function () {
+                        this.actor.role.data.victim = this.target;
+                    }
+                }
+            },
+            "Give To": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive"], exclude: ["dead", "self"] },
+                action: {
+                    labels: ["stealItem"],
+                    priority: PRIORITY_ITEM_GIVER_DEFAULT,
+                    run: function () {
+                        if (typeof this.actor.role.data.victim !== 'undefined')
```

Comment:
you can use a guard clause to reduce the code nesting and make it more readable

if victim is undefined OR the target alignment is Mafia {
  do nothing and return
}

stealItems()